public/celestia-app.sh

```diff
@@ -145,16 +145,16 @@ then
     echo ""
     echo "You can now run celestia-appd from anywhere." | tee -a "$LOGFILE"
     echo ""
-    echo "To check its version, you may execute the following command:" | tee -a "$LOGFILE"
+    echo "To check its version and see the menu, execute the following command:" | tee -a "$LOGFILE"
     echo ""
-    echo "celestia-appd version" | tee -a "$LOGFILE"
+    echo "celestia-appd version && celestia-appd --help" | tee -a "$LOGFILE"
 else
     echo ""
     echo "You can navigate to $TEMP_DIR to find and run celestia-appd." | tee -a "$LOGFILE"
     echo ""
-    echo "To run the app and check its version, you may execute the following commands:" | tee -a "$LOGFILE"
+    echo "To run the app and check its version and see the menu, execute the following commands:" | tee -a "$LOGFILE"
```

Comment:
[optional] since the commands below don't contain `celestia-appd start` I don't think this line needs to say "To run the app..."

```suggestion
    echo "To check its version and see the menu, execute the following commands:" | tee -a "$LOGFILE"
```